tests/test_invariant_Show-RestoreBackupDialog.py

```
@@ -0,0 +1,49 @@
+    ps_command = f'''
+    Import-Module "{script_path}" -Force
+    try {{
+        Load-RegistryBackupFromFile -FilePath "{payload}"
+        Write-Output "VALID"
+    }} catch {{
+        Write-Output "ERROR"
+    }}
+    '''
+    
+    # Execute the PowerShell command
+    result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)